optuna/storages/base.py

```diff
@@ -21,88 +21,87 @@ class BaseStorage(object, metaclass=abc.ABCMeta):
 
     This class is not supposed to be directly accessed by library users.
 
-    Storage classes abstract a backend database and provide library internal interfaces to
-    read/write history of studies and trials.
+    A storage class abstract a backend database and provide library internal interfaces to
```

Comment:
nit:
```suggestion
    A storage class abstracts a backend database and provide library internal interfaces to
```

Reply:
Ops, thanks for the catch. Let me apply your change.

Reply:
"provide" should probably be fixed too to "provides". Let me fix it.

Reply:
Fixed.